_guidelines-v4/09-textencoding/03-namesdates/02-namesdatesspecialized/04-namesdatespersnames.md

```diff
@@ -37,7 +37,7 @@ Personal names often consist of several components, such as given names, surname
 {:.gloss}
 **'genname' (generational name)**: contains a name component used to distinguish otherwise similar names on the basis of the relative ages or generations of the persons named.
 
-However, the recommended practice is to employ the following sub-elements provided the namesDates module:
+However, the recommended practice is to employ the following sub-elements provided by the namesDates module:
 
 {% include desc elem="addName" %}
 {% include desc elem="famName" %}
```

Comment:
In line 47:

The link to "model.nameLike" is broken (seems to be the case for other model links, too, cf. chapter 02). Is it necessary here at all?

Reply:
NB: Model links are resolved to "v4/elements/model.namelike.html" instead of "v4/model-classes/model.namelike.html"

Reply:
again, this is something not currently supported by `_includes/desc` and `_includes/desc-item`. Shouldn't be too hard to add, though. @ahankinson, @lpugin?